devel/ci/Dockerfile-rawhide

```diff
@@ -8,48 +8,80 @@ RUN dnf install --disablerepo rawhide-modular -y \
     findutils \
     git \
     liberation-mono-fonts \
+    python2-yaml \
+    python3-dnf \
+    python3-yaml \
     make \
+    python2-alembic \
+    python2-arrow \
+    python2-bleach \
+    python2-bugzilla \
     python2-click \
+    python2-colander \
+    python2-cryptography \
+    python2-dogpile-cache \
+    python2-fedmsg \
     python2-fedora \
+    python2-jinja2 \
+    python2-kitchen \
     python2-koji \
+    python2-librepo \
+    python2-markdown \
     python2-mock \
     python2-munch \
+    python2-openid \
+    python2-pillow \
+    python2-pylibravatar \
+    python2-pyramid \
+    python2-pyramid-mako \
     python2-pytest-cov \
+    python2-responses \
```

Comment:
Most of the ```python2-``` packages were removed purposefully in 64ff87102712acb89da99069eb1885dd1b64f29f because Rawhide doesn't have Bodhi's dependencies for Python 2 anymore (and Bodhi doesn't support Python 2 on Rawhide anymore either). Since the integration tests use Python 3, I'd think we don't really need these here, right?

It currently fails to build because ```python2-responses``` was removed from Rawhide.

Reply:
Yeah let's drop them.